README.md

```diff
@@ -46,3 +24,17 @@ As maintaining Variable groups in UI is tedious, all AKS pipeline variable group
 * You can get serviceEndpointId for keyvault variablegroup by running [Get Service End point Id script](variablegroups/scripts/get-service-end-point.sh) . If you don't have necessary privileges to list service connections, ask someone who has it. 
 * Refer [here](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate) for generating personal access token to authenticate Azure Devops.  
 
+### Updating Kubernetes version
+
+We regularly update the version of Kubernetes we run, and try to keep all clusters on the same version.
+
+The aim is to stay on the latest GA version of Kubernetes that Microsoft supports.
+
+We sometimes run the preview environment on newer versions so that we can try out new features and as a 'canary'
```

Comment:
Shall we clarify that before running preview on a newer version we test that version in sbox?